library/template_variables.tmpl

```diff
@@ -113,6 +113,32 @@
     {{- end -}}
 {{- end }}
 
+{{ define `include_models_in_unity_catalog` -}}
+    {{- if (eq .input_include_models_in_unity_catalog `no`) -}}
+        no
+    {{- else if (eq .input_include_models_in_unity_catalog `yes`) -}}
+        yes
+    {{- else -}}
+        {{ fail `Invalid selection of include_models_in_unity_catalog. Please choose from [no, yes]` }}
+    {{- end -}}
+{{- end }}
+
+{{ define `schema_name` -}}
+    {{- if (eq .input_include_models_in_unity_catalog `yes`) -}}
+        {{ .input_schema_name }}
```

Comment:
Could you add a validation to the `input_schema_name` when the UC models are used? Thanks. 
It would be in https://github.com/databricks/mlops-stack/blob/main/library/input_validation.tmpl
I'm not sure about the exact requirement but least `.` is not allowed in the name.

Reply:
good call, added